Makefile

```diff
@@ -30,7 +30,7 @@ fmt:
 .PHONY: fmt
 
 lint: check
-	./bin/golangci-lint run --disable godox --enable-all ./...
+	./bin/golangci-lint run --disable wsl --disable godox --enable-all ./...
```

Comment:
why disable wsl?

Reply:
because it causes a lot of errors and it is a new linter. I did not want to conflate it's issues with this MR